src/SFA.DAS.ApplyService.Data/ClarificationRepository.cs

```diff
@@ -20,6 +20,92 @@ public ClarificationRepository(IConfigurationService configurationService)
             _config = configurationService.GetConfig().GetAwaiter().GetResult();
         }
 
+        public async Task<ModerationOutcome> GetModerationOutcome(Guid applicationId, int sequenceNumber, int sectionNumber, string pageId)
+        {
+            using (var connection = new SqlConnection(_config.SqlConnectionString))
+            {
+                var moderationOutcomeResults = await connection.QueryAsync<ModerationOutcome>(
```

Comment:
you could rewrite this as connection.QueryFirstOrDefault I think? Not a big deal tho